UnityProject/Assets/Scripts/Systems/Spells/Spell.cs

```diff
@@ -165,6 +167,11 @@ IEnumerator DespawnAfterDelay()
 					var matrixInfo = MatrixManager.AtPoint(castPosition, true);
 					var localPos = MatrixManager.WorldToLocalInt(castPosition, matrixInfo);
 
+					if (matrixInfo.Matrix.Get<DoorMasterController>(localPos, true).Any(door => door.IsClosed))
```

Comment:
just add big old note next to it so if anyone looking for the code sees it instantly

Reply:
Forgot to commit it, anyways merge pls.